src/Turtle/Shell.hs

```diff
@@ -78,18 +79,22 @@ import Data.Monoid (Monoid(..), (<>))
 import Data.String (IsString(..))
 
 -- | A @(Shell a)@ is a protected stream of @a@'s with side effects
-newtype Shell a = Shell { foldIO :: forall r . FoldM IO a r -> IO r }
+newtype Shell a = Shell { _foldIO :: forall r . FoldM IO a r -> IO r }
+
+-- | Use a `FoldM IO` to reduce the stream of @a@'s produced by a `Shell`
```

Comment:
Put `foldIO` below `fold` to suggest to people `fold` is the more commonly used function